back/src/Guide/Api/Matomo.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+
+{- | Matomo curl requests -}
```

Comment:
`curl` is a tool for making requests. Calling all HTTP requests "curl requests" is not right.

```haskell
{- | Functions for interacting with Matomo (<https://matomo.org/>, our web analytics).
-}
```